modules/caddyhttp/serverless/testdata/pyechoserver/app.py

```
@@ -0,0 +1,73 @@
+        # Schedule shutdown after response is sent
+        logger.info(f"[{datetime.now().isoformat()}] Scheduling server shutdown")
+        import asyncio
+        asyncio.create_task(shutdown_after_delay())
+
+        logger.info(f"[{datetime.now().isoformat()}] Returning JSON response with status 200")
+        return JSONResponse(content=response_data, status_code=200)
+    except Exception as e:
+        error_timestamp = datetime.now().isoformat()
+        logger.error(f"[{error_timestamp}] Exception occurred: {str(e)}")
+        logger.error(f"[{error_timestamp}] Raising HTTPException with status 500")
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)